homework-submission/homework_week1/index.js

```diff
@@ -0,0 +1,80 @@
+// Create a todo list item as reusable component:
```

Comment:
Could you describe the props which the component accepts? What is the goal/effect of the prop?